src/app/ApplicationTemplate.Shared.Views/Framework/Review/ReviewService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.UI.Dispatching;
+
+namespace ApplicationTemplate;
+
+/// <summary>
+/// Implementation of <see cref="IReviewService"/>.
+/// </summary>
+public sealed class ReviewService : IReviewService
```

Comment:
Wasn't sure about this name. We also thought about something Native in the name.

Reply:
@jeanplevesque